### PR TITLE
Sanitize control characters in input str (#144)

### PR DESCRIPTION
**Pull request by marekmaskarinec**

### CODE REVIEW

**Comment by skejeton** [comment]
This will work for UI but I think it's actually better to sanitize it from within Tophat itself, because we don't really want inconsistent behavior across platforms.

File: umka/ui.um [hunk 1]
```diff
@@ -790,6 +790,13 @@ fn (gui: ^Gui) textBox*(tb: ^TextBox, cfg: TextBoxConfig = {}) {
 		}
 
 		runes := utf8.decode(input.getStr())
+		for i:=0; i < len(runes); i++ {
```
Comment (skejeton) [inline umka/ui.um:793]: Can we filter these out in tophat itself instead?

Comment (marekmaskarinec) [reply umka/ui.um:793]: That's a good idea. I will implement that.

**Comment by skejeton** [comment]
I'll test and merge if it works.

**Comment by skejeton** [comment]
Ok, it doesn't work but there's a simple reason, the problematic character is DEL, whose code is 127. I can fix that though. My main concern is, do we include `\n` and `\t` or do we handle it as `.enter`/`.tab` keys? For now I'm going to say the latter.

**Pull request merged**